backend/gn_plugin_depobio/tests/test_depobio.py

```diff
@@ -0,0 +1,205 @@
+        mock_validate.return_value = True
+
+        response_valid = self.client.get(
+            url_for(
+                "plugin_depobio.validate_folder_number",
+                folder_number=3391529,
+            )
+        )
+        assert response_valid.status_code == 200
+        assert response_valid.json
+
+    def get_folder(self, folder_number):
+        return self.client.get(
+            url_for(
+                "plugin_depobio.get_folder",
+                folder_number=folder_number,
+            )
+        )
+
+    @patch("gn_plugin_depobio.demarches_simplifiees.api.DemarchesSimplifieesConnexion.get_folder")
+    def test_get_folder_error(self, mock_get_folder, app, users_with_mail):
+        set_logged_user(self.client, users_with_mail["user"])
+        mock_get_folder.side_effect = TransportQueryError(
+            "a", errors=[{"extensions": {"code": ErrorCode.FORBIDDEN}}]
+        )
+        response_invalid = self.get_folder(99999999)
+        assert response_invalid.status_code == 403
+
+    @patch("gn_plugin_depobio.demarches_simplifiees.api.DemarchesSimplifieesConnexion.get_folder")
+    def test_get_folder(self, mock_get_folder, app, users_with_mail):
+        set_logged_user(self.client, users_with_mail["user"])
+        mock_get_folder.return_value = {
+            "id": "RG9zc2llci0zMzkxNTI5",
+            "number": 3391529,
+            "libelle": "Projet-test",
+        }
+        response_valid = self.get_folder(3391529)
+        assert response_valid.status_code == 200
+        folder = response_valid.json
+        assert folder["id"] == "RG9zc2llci0zMzkxNTI5"
+        assert folder["number"] == 3391529
+        assert folder["libelle"] == "Projet-test"
+
+
+@pytest.mark.usefixtures("client_class", "temporary_transaction")
+class TestMail:
+    def test_mail_builder(self, app, users_with_mail, acquisition_frameworks):
+        """
+        Test if the mail builded correspond to what we expect
+        """
+        set_logged_user(self.client, users_with_mail["user"])
+        af = acquisition_frameworks["af_1"]
+        # We need to simulate a request context because the mail builder use the current user
+        with app.test_request_context():
+            g.current_user = users_with_mail["stranger_user"]
+            mail_builder = MailBuilder(af)
+
+        assert mail_builder.mail["recipients"] == ["Stranger@example.com"]
+        assert (
+            mail_builder.mail["subject"]
+            == f"Dépôt du cadre d'acquisition {str(af.unique_acquisition_framework_id).upper()}"
+        )
+        assert "af_1" in mail_builder.mail["msg_html"]
+        assert str(af.unique_acquisition_framework_id).upper() in mail_builder.mail["msg_html"]
+
+
+class TestDSAPI:
+    @pytest.mark.skipif(
+        os.environ.get("GITHUB_ACTIONS") == "true",
+        reason="API not callable from CI Github because require white listed IP",
+    )
+    def test_validate_folder_number(self):
```

Comment:
```suggestion
    def test_validate_file_number_unmocked(self):
```

Pour ne pas avoir deux tests ayant le même nom - alternatives :
- Nommer le test présent "test_validate_file_number_unmocked" ou "test_validate_file_number_real_call" ou autre
- Nommer l'autre test ""test_validate_file_number_mocked" ou autre

Reply:
Je pense qu'il n'y à pas de problème, c'est deux tests différents puisque pas dans la même classe et donc pas de conflit de nommage